pyfar/signals/files.py

```diff
@@ -518,7 +516,10 @@ def _load_files(data):
 
     for file in files:
 
-        http_data = http.urlopen('GET', url + file)
```

Comment:
I think since we have ssl encryption for pyfar.org we don't need to ignore any warnings, we can just change the PoolManager to http = urllib3.PoolManager(cert_reqs='CERT_REQUIRED')

Reply:
Thanks for the hint. That works.